lib/usd/translators/particleWriter.h

```diff
@@ -17,27 +17,23 @@
 #define PXRUSDTRANSLATORS_PARTICLE_WRITER_H
 
 /// \file pxrUsdTranslators/particleWriter.h
+#include <mayaUsd/fileio/transformWriter.h>
```

Comment:
Blank line between Doxygen tag and first include

Reply:
See c6eecc5